grails-app/domain/org/transmartproject/db/dataquery/highdim/DeGplInfo.groovy

```diff
@@ -35,7 +35,7 @@ class DeGplInfo implements Platform {
         table         schema: 'deapp'
 
         id              column: 'platform',   generator: 'assigned'
-        genomeReleaseId column: 'release_nbr'
+        genomeReleaseId column: 'genome_build'
 
```

Comment:
@forus GenomeReleaseId is defined in transmart-core-api (Platform.groovy). I don't want to make changes that divert from this API.

@peterrice Two related pull requests (https://github.com/transmart/transmart-data/pull/61 and https://github.com/transmart/tranSMART-ETL/pull/8) have been merged this afternoon. This pull request is also a part of that update and should for consistency reasons also merged.